substrate/frame/treasury/src/lib.rs

```diff
@@ -1120,3 +1122,15 @@ impl<T: Config<I>, I: 'static> OnUnbalanced<NegativeImbalanceOf<T, I>> for Palle
 		Self::deposit_event(Event::Deposit { value: numeric_amount });
 	}
 }
+
+/// TypedGet implementaion to get the AccountId of the Treasury.
+pub struct TreasuryAccountId<R>(PhantomData<R>);
+impl<R> sp_runtime::traits::TypedGet for TreasuryAccountId<R>
+where
+	R: crate::Config + frame_system::Config,
```

Comment:
Isn't the `crate::Config` already a `frame_system::Config`?

Reply:
Yes, good catch :)